components/kyma-environment-broker/internal/process/provisioning/busola_migrator_overrides.go

```diff
@@ -0,0 +1,38 @@
+package provisioning
+
+import (
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/sirupsen/logrus"
+)
+
+const BusolaMigratorComponentName = "busola-migrator"
+
+type BusolaMigratorOverridesStep struct {
+}
+
+func NewBusolaMigratorOverridesStep() *BusolaMigratorOverridesStep {
+	return &BusolaMigratorOverridesStep{}
+}
+
+func (s *BusolaMigratorOverridesStep) Name() string {
+	return "BusolaMigratorOverrides"
+}
+
+func (s *BusolaMigratorOverridesStep) Run(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
+	kubeconfigIDOverrides := []*gqlschema.ConfigEntryInput{
+		{
+			Key:   "deployment.env.kubeconfigID",
+			Value: operation.InstanceID,
+		},
+		{
+			Key:   "global.istio.gateway.name",
```

Comment:
I think this global override should be already applied during the Kyma installation, are you sure we need it?

Reply:
Yes, it wasn't applied during my initial testing.

Reply:
This override is not included in the `global-overrides-*` configmaps. I am not sure if we should include here or add it to the these configmap.